client/landing/gutenboarding/components/signup-form/style.scss

```diff
@@ -1,26 +1,71 @@
+@import '../../variables.scss';
 @import '~@automattic/calypso-color-schemes/src/calypso-color-schemes';
+@import 'assets/stylesheets/gutenberg-base-styles';
+@import '../../mixins';
+
+.signup-form__overlay.components-modal__screen-overlay {
+	background-color: $white;
+}
 
 .signup-form.components-modal__frame {
 	border: none;
-	border-radius: 3px;
-	min-width: 450px;
+	border-radius: 0;
+	box-shadow: none;
+	top: 0;
+	left: 0;
+	right: auto;
+	bottom: auto;
+	width: 100%;
+	height: 100%;
+	max-width: 100%;
+	max-height: 100%;
+	transform: translate( 0%, 0% );
 
 	.components-modal__header {
-		.components-modal__header-heading {
-			font-size: 20px;
-			font-weight: normal;
-		}
+		display: none;
 	}
-	.components-text-control__input {
-		padding: 7px 14px;
-		font-size: 16px;
-		line-height: 1.5;
+
+	.components-modal__content {
+		padding: 0;
 	}
 
-	.signup-form__terms-of-service-link {
+	.signup-form__title {
+		@include onboarding-heading-text-mobile;
+		text-align: center;
+
+		@include break-mobile {
+			@include onboarding-heading-text;
+		}
+	}
+
+	.signup-form__body {
+		padding: 0 20px 20px;
+		position: absolute;
+		top: 50%;
+		left: 50%;
+		width: 100%;
+		max-width: 500px;
+		transform: translate( -50%, -50% );
 		text-align: center;
-		color: var( --color-text-subtle );
-		margin: 20px 0 40px;
+	}
+
+	.signup-form__legend p {
+		@include onboarding-medium-text;
+		margin: 15px 0 30px;
+		color: var( --studio-gray-40 );
+	}
+
+	.components-text-control__input {
+		padding: 21px 14px;
+		@include onboarding-medium-text;
+		border-radius: 0;
+		border: 1px solid var( --studio-gray-10 );
+		width: 100%;
+		max-width: 100%;
+
+		@include break-mobile {
+			max-width: 400px;
+		}
 	}
 
 	.signup-form__error-notice {
```

Comment:
My 2c on the error notice design: changing this rule to `margin: 0 30px` on desktop (and something else on mobile) is probably enough. Makes it line up with the text input, but that's it.

Reply:
GH hasn't made it super clear, but I'm referring to the `.signup-form__error-notice` rules.

Reply:
Great idea, that looks so much nicer.